File: flowcamera/src/main/java/com/hbzhou/open/flowcamera/listener/CaptureListener.java
package com.hbzhou.open.flowcamera.listener;

/**
 * author hbzhou
 * date 2019/12/13 10:49
 */

public interface CaptureListener {
    void takePictures();

    void recordShort(long time);

    void recordStart();

    void recordEnd(long time);

    void recordZoom(float zoom);

    void recordError();
}

File: flowcamera/src/main/java/com/hbzhou/open/flowcamera/CaptureListener.kt
package com.hbzhou.open.flowcamera


/**
 * author hbzhou
 * date 2019/12/13 11:13
 */
interface CaptureListener {

    fun takePictures()

    fun recordShort(time: Long)

    fun recordStart()

    fun recordEnd(time: Long)

    fun recordZoom(zoom: Float)

    fun recordError()
}